specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -1052,6 +1052,56 @@
         }
       }
     },
+    "/routing/jobs/{jobId}/assignments/{assignmentId}:unassign": {
+      "post": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Un-assign a job.",
+        "operationId": "JobRouter_UnassignJobAction",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
```

Comment:
Might be good to outsource the parameter and ref it here.

Reply:
Leaving them here to be consistent with rest of the spec for string parameters.